src/Cabinet/Cabinet.java

```diff
@@ -0,0 +1,144 @@
+package Cabinet;
+
+import Comparatori.NrPacientiComparator;
+import Comparatori.NotaComparator;
+import Personal.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Cabinet {
+    private ArrayList<Doctor> doctori;
+    private ArrayList<Asistent> asistenti;
+    HashMap<String, Integer> stoc;
```

Comment:
declare it as Map<String, Integer> and on line 23 since you put a HashMap it will be fine due to polimorfism :) we usually use the abstract type for references